src/coreclr/src/ToolBox/superpmi/superpmi-shared/icorjitinfoimpl.h

```diff
@@ -1,7 +1,7 @@
-//
-// Copyright (c) Microsoft. All rights reserved.
-// Licensed under the MIT license. See LICENSE file in the project root for full license information.
-//
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+// DO NOT EDIT THIS FILE! It IS AUTOGENERATED
```

Comment:
Can you leave breadcrumbs here and in the other newly auto-generated files to hiht at how they can be regenerated?

Reply:
Yes, please include pointers to the instructions for re-generation

Reply:
https://github.com/dotnet/runtime/blob/master/docs/project/updating-jitinterface.md is the current doc if changes are needed there.